packages/grid_client/src/clients/tf-grid/contracts.ts

```diff
@@ -76,7 +133,8 @@ class TFContracts extends Contracts {
         rentContractsCount: rentContractsCount,
       });
 
-      return response["data"];
+      const data = response["data"] as GqlContracts;
+      return data;
```

Comment:
We are not using the data variable anywhere; we can return its value directly.
```suggestion
    return response["data"] as GqlContracts;
```